README.md

```diff
@@ -320,6 +323,34 @@ tube_evaluator = TubeEvaluator(annot_filepath, preds_filepath)
 res, mAP = tube_evaluator.evaluate(thr=0.5)
 ```
 
+### Running CLI
+
+If there is a need for a command line interface (CLI), you can run the `cli.py` file.
+
+#### CLI Arguments
+The following table shows all the possible arguments you can provide to the CLI:
+| Argument        | Description                                                                                                                                           | Example                                                      | Default | Notes                                                                                                                  |
+|-----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------|---------|------------------------------------------------------------------------------------------------------------------------|
+| --help, -h      | Shows a help message for tool                                                                                                                         | python cli.py -h                                             |         |                                                                                                                        |
+| --anno_gt       | Takes in a directory or file to the ground-truth annotations                                                                                          | python cli.py --anno_gt toyexample/gts_vocpascal_format      |         | Based on the `gtformat`, make sure this parameter points to the right kind of ground truth (directory or file)         |
+| --anno_gt       | Takes in a directory or file to the detection annotations                                                                                             | python cli.py --anno_gt toyexample/dets_classname_rel_xyx2y2 |         | Based on the `detformat`, make sure this parameter points to the right kind of ground truth (directory or file)        |
+| --img           | Takes in a directory where the corresponding images are located                                                                                       | python cli.py --img toyexample/images                        |         | Used in some ground truth annotation formats. At the current moment, there is visualization functionality for the CLI  |
```

Comment:
Shouldn't it be ".... there is no visualization functionality for the CLI." ?

Reply:
Yup, thank you for catching that typo!

Reply:
Fixed in https://github.com/rafaelpadilla/review_object_detection_metrics/pull/57/commits/27394185d27d073b2f871dbd7e836ee038c16ad9